lib/vox/builder/file.ex

```diff
@@ -1,6 +1,17 @@
 defmodule Vox.Builder.File do
   @src_dir Application.compile_env(:vox, :src_dir)
-
+  @type t :: %__MODULE__{
+          bindings: keyword(),
+          collections: list(),
+          content: binary(),
+          destination_path: binary(),
+          final: binary(),
+          root_template: binary(),
+          source_path: binary(),
+          template: binary(),
+          compiled: any(),
```

Comment:
what about compiled type

Reply:
`compiled` stores the result of running `EEx.compile_file/2` which [according to the source](https://github.com/elixir-lang/elixir/blob/9fd97c466301baa74b309f3ea62215a12911c4d8/lib/eex/lib/eex.ex#L255), returns a `Macro.t()`. So
```suggestion
          compiled: Macro.t(),
```